fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/CheckWalSizeAction.java

```diff
@@ -0,0 +1,113 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.httpv2.rest;
+
+import org.apache.doris.catalog.Env;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.DdlException;
+import org.apache.doris.httpv2.entity.ResponseEntityBuilder;
+import org.apache.doris.mysql.privilege.PrivPredicate;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.system.Backend;
+import org.apache.doris.system.SystemInfoService;
+import org.apache.doris.system.SystemInfoService.HostInfo;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * cal wal size of specific be
+ * fe_host:fe_http_port/api/get_wal_size?host_ports=host:port,host2:port2...
+ * return:
+ * {
+ * "msg": "OK",
+ * "code": 0,
+ * "data": ["192.168.10.11:9050:1", "192.168.10.11:9050:0"],
+ * "count": 0
+ * }
+ */
+
+@RestController
+public class CheckWalSizeAction extends RestBaseController {
+    public static final String HOST_PORTS = "host_ports";
+
+    @RequestMapping(path = "/api/get_wal_size", method = RequestMethod.GET)
+    public Object execute(HttpServletRequest request, HttpServletResponse response) {
+        // check user auth
+        executeCheckPassword(request, response);
+        checkGlobalAuth(ConnectContext.get().getCurrentUserIdentity(), PrivPredicate.OPERATOR);
+
+        String hostPorts = request.getParameter(HOST_PORTS);
+        if (Strings.isNullOrEmpty(hostPorts)) {
```

Comment:
Using Guava's Strings utility class for checking null or empty strings is fine,
but Spring has built-in mechanisms for request parameter validation that could be used here.